packages/css/src/plugins/atomicify-rules.ts

```diff
@@ -151,6 +151,47 @@ const atomicifyRule = (node: Rule, opts: AtomicifyOpts): Rule[] => {
     .filter((child): child is Rule => !!child);
 };
 
+/**
+ * Checks whether the given at-rule node can be
+ * atomicified (transformed into atomic rules).
+ *
+ * Throws an error for unknown at-rules, as well as
+ * at-rules that should not be used in the stylesheet.
+ *
+ * @param node
+ */
+const canAtomicifyAtRule = (node: AtRule): boolean => {
+  const canBeAtomificied = [
+    'container',
+    '-moz-document',
+    'else',
+    'layer',
+    'media',
+    'supports',
+    'when',
+  ];
+  const forbidden = ['charset', 'import', 'namespace'];
+  const ignored = [
+    'color-profile',
+    'counter-style',
+    'font-face',
+    'font-palette-values',
+    'keyframes',
```

Comment:
Can we handle `keyframes` as a special case?

Instead of ignoring it, I suggest we throw an error. The error is something like
```
You cannot use @keyframes at-rule here. Please see the [documentation](https://compiledcssinjs.com/docs/api-keyframes).
```

Reply:
Sure thing!

Reply:
Are there any other at-rules for which we have specific recommendations in the form of documentation (or some other external-facing document)? I'm happy to add some more specific errors for different at-rules if so

Reply:
Not that I am aware of.

Reply:
Did some digging into this, and unfortunately I wasn't able to find a way to throw an error for `@keyframes` usage here without breaking [our own keyframes API](https://compiledcssinjs.com/docs/api-keyframes) at the same time. This is because keyframes({...}) and keyframes\`...\` [are converted](https://github.com/atlassian-labs/compiled/blob/14f2091b30287c21b618ab4b3c26aec1f12f9ab8/packages/babel-plugin/src/utils/css-builders.ts#L573) to `@keyframes ... { ... }` before the contents of `atomicify-rules.ts` is run.

Will mark this suggestion as out-of-scope (maybe something for another PR) if you're ok with this @liamqma

Reply:
Sounds good. Thanks for looking into it.